tests/integ/modin/series/test_dt_accessor.py

```diff
@@ -31,6 +31,7 @@
         "is_quarter_end",
         "is_year_start",
         "is_year_end",
+        "is_leap_year",
```

Comment:
I don't see a test case testing `dt_properties` with leap years. Is there one?

Reply:
Yes, `test_dt_properties` has a line:
```
native_pd.date_range(start="2021-01-01", periods=5, freq=freq)
```
and one of the values for `freq` is `y`. So you end up with 5 consecutive years including a leap year.